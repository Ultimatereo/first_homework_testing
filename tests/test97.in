5
83077684592 -684072012194 -40660667433 -437881021833 -996161635212 642680969951
107920317387 -814185963373 -909175533800 -872915255913 318675371739 -809990310312
-735457645421 -20455142085 -688570653510 589679531289 218396564154 -540580689713
-606123432834 -29811324978 265494550472 700656533329 247347261779 93066121901
-177840448863 433882829916 760010588280 -280496699905 -218772251002 948778620486
