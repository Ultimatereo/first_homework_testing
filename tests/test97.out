-0.429818
0.121851
0.777945
-0.292892
-0.667687
